src/metabase/api/util.clj

```diff
@@ -2,6 +2,7 @@
   (:require [compojure.core :refer [defroutes GET POST]]
             [metabase.api.common :refer :all]
             [metabase.logger :as logger]
+            [metabase.util.stats :as stats]
             [metabase.util.schema :as su]))
```

Comment:
Can you make sure you keep these alphabetized?

https://github.com/metabase/metabase/wiki/Metabase-Clojure-Style-Guide#keep-imports-ordered